src/components/app/routes/queries/canRedeemCourse.js

```diff
@@ -11,7 +10,7 @@ import { getAvailableCourseRuns } from '../../../course/data/utils';
  * @param {array} courseRunKeys List of course run keys.
  * @returns Promise for get request from the authenticated http client.
  */
-const fetchCanRedeem = async (enterpriseId, courseRunKeys) => {
+export const fetchCanRedeem = async (enterpriseId, courseRunKeys) => {
```

Comment:
[clarification] I think the directionality of the refactor might be a bit backwards here? That is, you've created a new `services.js` file, however, its contents are not service-related functions (e.g., `fetchCanRedeem`). The service-related function is actually remaining in this `queries` specific directory. Would it be possible to flip this (hopefully without running into dependency cycle issue) where the `fetch*` related functions are defined in the services file but the helper functions to access the query key factory kept in this `queries` directory?

(Note: comment applies broadly, not just to this one function).